packages/image_picker/image_picker_for_web/example/test/readme_excerpts_test.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:image_picker_for_web_integration_tests/readme_excerpts.dart';
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+
+void main() {
+  testWidgets('getImageFromPath loads image from XFile path',
+      (WidgetTester tester) async {
+    // Create an XFile using the test image path.
+    final XFile pickedFile = XFile('assets/flutter-mark-square-64.png');
```

Comment:
IIRC, you can't read asset files in tests on web, and this would need to be a blob instead on web. @ditman Am I remembering that correctly?

Reply:
Yes, this is correct. Here's a couple of examples of `XFile`s in tests:

* https://github.com/flutter/packages/blob/main/packages/cross_file/test/x_file_html_test.dart
* https://github.com/flutter/packages/blob/main/packages/file_selector/file_selector_web/example/integration_test/file_selector_web_test.dart#L134-L137